ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -43,12 +61,57 @@ const PersonalSignInfo: React.FC = () => {
     <>
       {isSIWE && useTransactionSimulations && (
         <ConfirmInfoSection>
-          <ConfirmInfoRow
-            label={t('simulationDetailsTitle')}
-            tooltip={t('simulationDetailsTitleTooltip')}
+          <Box
```

Comment:
I assume the necessary formatting isn't feasible to encapsulate within the `ConfirmInfoRow` component?